examples/skr/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,55 @@
+# Troubleshooting Guide for Deploying the SKR Sidecar
+
+## 401 Unauthorized Error
+
+When running the importkey tool, you may see the following error:
+
+```
+pulling AKV response body failed: http response status equal to 401 Unauthorized
+```
+
+To resolve this issue, generate a new bearer token and copy it into the importkeyconfig.json.
```

Comment:
We can skip the boilerplate "To resolve this issue"... 
Also, generate a new bearer <From where>?